nodestream/project/run_request.py

```diff
@@ -17,6 +17,7 @@ class RunRequest:
     """
 
     pipeline_name: str
+    pipeline_scope: str
```

Comment:
At first glance this seems but fine. It definitely gets the job done. But I'm troubled by `RunRequest`  getting `pipeline_scope`. 

If the request is meant to model what the user asked to have run, then have a `pipeline_scope` fields feels like it should be a filter or specifier and not just metadata that is being carried forward.  Feels like this is emblematic of an encapsulation problem.